<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="SANISU2" name="Test" tests="16" failures="8" timestamp="16 Feb 2016 21:41:24 GMT" time="0.078" errors="0">
  <testcase name="HashmapgetvaluefromKey" time="0.012" classname="com.test.HashMapDemoTest"/>
  <testcase name="Hashmapgetvaluefromkeneg" time="0.001" classname="com.test.HashMapDemoTest">
    <failure type="java.lang.AssertionError" message="expected:&amp;lt;nishitha&amp;gt; but was:&amp;lt;smitha&amp;gt;">
      <![CDATA[java.lang.AssertionError: expected:<nishitha> but was:<smitha>
	at com.test.HashMapDemoTest.Hashmapgetvaluefromkeneg(HashMapDemoTest.java:50)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- Hashmapgetvaluefromkeneg -->
  <testcase name="findLargestNumber" time="0.002" classname="com.test.LargestNumberTest"/>
  <testcase name="findLargestNumberneg" time="0.001" classname="com.test.LargestNumberTest">
    <failure type="java.lang.AssertionError" message="expected:&amp;lt;98&amp;gt; but was:&amp;lt;63&amp;gt;">
      <![CDATA[java.lang.AssertionError: expected:<98> but was:<63>
	at com.test.LargestNumberTest.findLargestNumberneg(LargestNumberTest.java:27)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- findLargestNumberneg -->
  <testcase name="findPalindrome" time="0.002" classname="com.test.PalindromeTest"/>
  <testcase name="findPalindromeneg" time="0.001" classname="com.test.PalindromeTest">
    <failure type="java.lang.AssertionError" message="expected:&amp;lt;false&amp;gt; but was:&amp;lt;true&amp;gt;">
      <![CDATA[java.lang.AssertionError: expected:<false> but was:<true>
	at com.test.PalindromeTest.findPalindromeneg(PalindromeTest.java:24)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- findPalindromeneg -->
  <testcase name="Wordreversetest" time="0.002" classname="com.test.WordReverseTest"/>
  <testcase name="Wordreversetestneg" time="0.001" classname="com.test.WordReverseTest">
    <failure type="java.lang.AssertionError" message="expected:&amp;lt; This  is  the  string  which  is reveresed&amp;gt; but was:&amp;lt; This is the string which is reveresed&amp;gt;">
      <![CDATA[java.lang.AssertionError: expected:< This  is  the  string  which  is reveresed> but was:< This is the string which is reveresed>
	at com.test.WordReverseTest.Wordreversetestneg(WordReverseTest.java:26)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- Wordreversetestneg -->
  <testcase name="wordReplaceneg" time="0.004" classname="com.test.WordReplaceTest">
    <failure type="java.lang.AssertionError" message="expected:&amp;lt;true&amp;gt; but was:&amp;lt;false&amp;gt;">
      <![CDATA[java.lang.AssertionError: expected:<true> but was:<false>
	at com.test.WordReplaceTest.wordReplaceneg(WordReplaceTest.java:24)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- wordReplaceneg -->
  <testcase name="wordReplacepos" time="0.002" classname="com.test.WordReplaceTest"/>
  <testcase name="Methodoverloadingneg" time="0.001" classname="com.test.InheritanceTest">
    <failure type="java.lang.AssertionError" message="expected:&amp;lt;13&amp;gt; but was:&amp;lt;12&amp;gt;">
      <![CDATA[java.lang.AssertionError: expected:<13> but was:<12>
	at com.test.InheritanceTest.Methodoverloadingneg(InheritanceTest.java:21)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- Methodoverloadingneg -->
  <testcase name="Methodoverloadingpos" time="0.001" classname="com.test.InheritanceTest"/>
  <testcase name="Methodoverridingneg" time="0.001" classname="com.test.InheritanceTest"/>
  <testcase name="Methodoverridingpos" time="0.0" classname="com.test.InheritanceTest">
    <failure type="java.lang.AssertionError" message="expected:&amp;lt;35&amp;gt; but was:&amp;lt;12&amp;gt;">
      <![CDATA[java.lang.AssertionError: expected:<35> but was:<12>
	at com.test.InheritanceTest.Methodoverridingpos(InheritanceTest.java:31)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- Methodoverridingpos -->
  <testcase name="ArrayIndexOutofBound" time="0.002" classname="com.test.NullpointerTest"/>
  <testcase name="Nullpointerpos" time="0.001" classname="com.test.NullpointerTest">
    <failure type="org.testng.TestException" message="
Method NullpointerTest.Nullpointerpos()[pri:0, instance:com.test.NullpointerTest@37f92637] should have thrown an exception of type class java.lang.ArithmeticException">
      <![CDATA[org.testng.TestException: 
Method NullpointerTest.Nullpointerpos()[pri:0, instance:com.test.NullpointerTest@37f92637] should have thrown an exception of type class java.lang.ArithmeticException
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- Nullpointerpos -->
</testsuite> <!-- Test -->
